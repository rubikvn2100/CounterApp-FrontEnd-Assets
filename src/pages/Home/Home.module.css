.homepage {
  height: 100vh;
  background-image: linear-gradient(to bottom, #0060ff0f 0%, #ffffff 70%);
}
