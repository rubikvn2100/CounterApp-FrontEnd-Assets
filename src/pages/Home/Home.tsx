import React from "react";
import styles from "./Home.module.css";

const Home: React.FC = () => {
  return (
    <div className={styles.homepage}>
      <div>Dummy Home Page</div>
    </div>
  );
};

export default Home;
