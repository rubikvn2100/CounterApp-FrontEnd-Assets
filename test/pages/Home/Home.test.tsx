import React from "react";
import renderer from "react-test-renderer";
import Home from "../../../src/pages/Home/Home";

describe("Home component", () => {
  test("matches the snapshot", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
