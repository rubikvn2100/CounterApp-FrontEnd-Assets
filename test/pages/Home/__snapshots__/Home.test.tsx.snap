// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches the snapshot 1`] = `
<div>
  <div>
    Dummy Home Page
  </div>
</div>
`;
